pkgs/cupertino_http/lib/src/cupertino_client.dart

```diff
@@ -17,6 +17,26 @@ final _digitRegex = RegExp(r'^\d+$');
 
 const _nsurlErrorCancelled = -999;
 
+/// A [ClientException] generated from an [NSError].
+class NSErrorClientExceptions extends ClientException {
```

Comment:
Looks like you have a typo in the class name. Meant to be `NSErrorClientException`?

Reply:
Thanks! Fixed.